bin/signalk-server

```diff
@@ -16,6 +16,16 @@
  * limitations under the License.
 */
 
+const { engines } = require('../package.json');
+const semver = require('semver');
+
+const version = engines.node;
+
+if (!semver.satisfies(process.version, version)) {
```

Comment:
This is a nice idea in theory, but in practise the minimum is now 16 (we are running tests with it so we don't accidentally mess that up).

I think this should
- !== 18 => log "The recommended version of Node.js is 18, you are running ${process.version}"
- <16  => error log what this does now

Reply:
Yep, was thinking about the same thing. Docker images already use 18.

But going forward we’ll probably still have minimum separate from recommended that would be latest lts.

Reply:
#1584